config/webpack.dev.config.js

```diff
@@ -68,43 +107,19 @@ module.exports = merge(commonConfig, {
       // flash-of-unstyled-content issues in development.
       {
         test: /(.scss|.css)$/,
-        use: [
-          'style-loader', // creates style nodes from JS strings
+        oneOf: [
           {
-            loader: 'css-loader', // translates CSS into CommonJS
-            options: {
-              sourceMap: true,
-              modules: {
-                compileType: 'icss',
-              },
-            },
-          },
-          {
-            loader: 'postcss-loader',
-            options: {
-              postcssOptions: {
-                plugins: [
-                  PostCssAutoprefixerPlugin(),
-                  PostCssRTLCSS(),
-                  PostCssCustomMediaCSS(),
-                ],
-              },
-            },
+            resource: /(@openedx\/paragon|@edx\/brand)/,
```

Comment:
nit: if other code paths are supporting `@openedx/brand` as the NPM alias (which afaik MFEs do not yet use), does this regex support `@openedx/brand`, too? It currently only supports `@edx/brand` while other code paths only supports `@openedx/brand`.

Reply:
This is solve @adamstankiewicz @brian-smith-tcril